core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -267,8 +267,6 @@ angular.module('oppia').component('explorationEditorPage', {
             explorationData.auto_tts_enabled);
           ExplorationCorrectnessFeedbackService.init(
             explorationData.correctness_feedback_enabled);
-          StateClassifierMappingService.init(
```

Comment:
Looks like we are no longer using this service in this file. In that case, let's remove StateClassifierMappingService dependency from this file.

Reply:
Done